### ISSUE TITLE
Can I directly train swin v2 large on imagenet 1k ?

### ISSUE THREAD

**Issue by Zhaoyi-Yan** [microsoft/Swin-Transformer#261]
If I directly train swin v2-L on imagenet 1k, then will it perform better than swin v2-B trained on imagenet 1k ?

**Comment by ancientmooner** [comment]
> 

No, it will not get better accuracy. But if you use SimMIM pre-training, Swin V2-L will perform better than Swin V2-B. Please try https://github.com/microsoft/Swin-Transformer/blob/main/get_started.md#simmim-support
